projects/default/controllers/ros/Ros.cpp

```diff
@@ -447,7 +447,7 @@ void Ros::run(int argc, char **argv) {
     ros::spinOnce();
     publishClockIfNeeded();
     for (unsigned int i = 0; i < mSensorList.size(); i++)
-      mSensorList[i]->publishValues(mStep * mStepSize);
+      mSensorList[i]->publishValues(mStep * mRobot->getBasicTimeStep());
```

Comment:
The `mStepSize` is configured in the `timeStepCallback` function. To preserve the functionality of the `timeStepCallback`, I would rather set default value of the `mStepSize` to `mRobot->getBasicTimeStep()` as soon as we have access to the `Robot` object.